geode-core/src/integrationTest/java/org/apache/geode/distributed/LocatorLauncherLocalIntegrationTest.java

```diff
@@ -74,6 +77,16 @@ public void startReturnsOnline() {
     assertThat(launcher.start().getStatus()).isEqualTo(ONLINE);
   }
 
+  @Test
+  public void whenLocatorIsPresetThenItWillNotReadThePropertyFiles() {
+    launcher = givenLocatorLauncher();
+
+    assertThat(launcher.start().getStatus()).isEqualTo(ONLINE);
+    LocatorLauncher locatorLauncher = spy(launcher);
+    locatorLauncher.status();
+    verify(locatorLauncher, times((0))).getProperties();
```

Comment:
I wish this test was more direct. What we really want to verify is that no `DistributionConfigImpl` is constructed.

I understand that's not feasible with Assert-J. So I recommend a comment like this:

```java
/*
 What we really want to verify is that no new DistributionConfigImpl is constructed.
 But that's had/impossible to verify with Assert-J so we're approximating that check.
 See getSslConfig(), which calls getProperties() in situations where a new
 DistributionConfigImpl is constructed.
*/
```